_posts/2017-10-12-windows-retrospect.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: posts
+title: Bazel on Windows: a year in retrospect
+---
+
+Bazel on Windows is no longer experimental. We think Bazel on Windows is now
+stable and usable enough to qualify for the Beta status we have given Bazel on
+other platforms.
+
+Over the last year, we put a lot of work into improving Bazel's Windows support:
+
+*   **Bazel no longer depends on the MSYS runtime.** This means Bazel is now a
+    native Windows binary and we no longer link it to the MSYS DLLs. Bazel still
+    needs Bash (MSYS or Git Bash) and the GNU binutils (binaries under
+    `/usr/bin`) if your dependency graph includes `genrule` or `sh_*` rules
+    (similarly to requiring `python.exe` to build `py_*` rules), but you can use
+    any MSYS version and flavour you like, including Git Bash.
+*   **Bazel is easier to set up.** You now (typically) no longer need to set the
+    following environment variables:
+    *   `BAZEL_SH` and `BAZEL_PYTHON` -- Bazel attempts to autodetect your MSYS
```

Comment:
use BASH rather than MSYS since MSYS is no longer needed.

Reply:
Done.